spec/features/consumer/registration_spec.rb

```diff
@@ -6,7 +6,20 @@
 
   describe "Registering a Profile" do
     let(:user) { create(:user, password: "password", password_confirmation: "password") }
-    before { Spree::Config.enterprises_require_tos = false }
+
+    before do
+      Spree::Config.enterprises_require_tos = false
+
+      albania = Spree::Country.create!({ name: "Albania", iso3: "ALB", iso: "AL", iso_name: "ALBANIA", numcode: "8" }, without_protection: true)
+      Spree::State.create!({ name: "Berat", abbr: "BRA", country: albania }, without_protection: true)
+      Spree::Country.create!({ name: "Chad", iso3: "TCD", iso: "TD", iso_name: "CHAD", numcode: "148" }, without_protection: true)
+    end
+
+    after do
+      Spree::State.where(name: 'Berat').delete_all
+      Spree::Country.where(name: 'Albania').delete_all
+      Spree::Country.where(name: 'Chad').delete_all
+    end
```

Comment:
Don't we use something like https://github.com/DatabaseCleaner/database_cleaner ? Do we always cleanup the data manually after the test?